serverless-configs/aws-functions.yml

```diff
@@ -686,6 +686,21 @@
             type: request
             resultTtlInSeconds: 0
 
+
+  adminGetPreferencesMetrics:
+    handler: serverless/src/adminGetPreferencesMetrics/handler.default
+    timeout: ${env:LAMBDA_TIMEOUT, '30'}
+    events:
+      - http:
+          method: get
+          cors: ${file(./serverless-configs/${self:provider.name}-cors-configuration.yml)}
+          path: admin/preferencesMetrics
```

Comment:
I'm not sure why, but all the URLs for our API Gateway use underscores, not hyphens. But this `path` should match all the others